src/_includes/zebra/radio--activities.html

```diff
@@ -0,0 +1,86 @@
+<!--
+  Variation from default:
+  - Different content
+  - Added required attribute and legend text
+  - Added aria-disabled option with function to prevent default behavior
+
+  Could be reusable if we need aria-disabled pattern
+-->
+
+<!--start radio activities-->
+<fieldset class="usa-fieldset">
+  <legend class="usa-legend">
+    Choose the activity you want to reserve (select one option):
+    <abbr title="required" class="usa-hint usa-hint--required">*</abbr>
+  </legend>
+  <div class="usa-radio">
+    <input
+      class="usa-radio__input usa-radio__input--tile"
+      id="coyote-camp"
+      type="radio"
+      name="activity"
+      value="coyotes"
+      required
+    />
+    <label class="usa-radio__label" for="coyote-camp">
+      Camp among the coyotes
+    </label>
+  </div>
+  <div class="usa-radio">
+    <input
+      class="usa-radio__input usa-radio__input--tile"
+      id="buffalo-fly"
+      type="radio"
+      name="activity"
+    />
+    <label class="usa-radio__label" for="buffalo-fly">
+      Fly over buffalo valley
+    </label>
+  </div>
+  <div class="usa-radio">
+    <input
+      class="usa-radio__input usa-radio__input--tile"
+      id="secret-hike"
+      type="radio"
+      name="activity"
+      value="hike"
+    />
+    <label class="usa-radio__label" for="secret-hike">
+      Hike the secret trail
+    </label>
+  </div>
+  <div class="usa-radio">
+    <input
+      class="usa-radio__input usa-radio__input--tile"
+      id="salmon-bear"
+      type="radio"
+      name="activity"
+      value="bear"
+    />
+    <label class="usa-radio__label" for="salmon-bear">
+      Watch bear hunt for salmon
+    </label>
+  </div>
+  <div class="usa-radio">
+      <input
+        class="usa-radio__input usa-radio__input--tile"
+        id="atv-buffalo"
+        type="radio"
+        name="activity"
+        value="atv"
+        aria-disabled="true"
+        onclick="preventClick(event);"
+      />
+      <label class="usa-radio__label" for="atv-buffalo" style="pointer-events: none;">
```

Comment:
Important catch! Thanks @mahoneycm. It was needed for an earlier iteration but has a negative impact now. 

Removed in [6c895b7](https://github.com/uswds/uswds-sandbox/pull/126/commits/6c895b7d50cc334b8758805eec4cd9f0478f7c99)

cc: @jaclinec: Alerting you that this will make a slight change from the pilot versions. We are making this change to move back to the default USWDS "disabled" cursor behavior (which @mahoneycm has illustrated in the comment above). Let me know if you need this reversed.